packages/massmapper-app/src/services/MapService.ts

```diff
@@ -196,11 +201,36 @@ class MapService {
 			toAdd.length > 0 && r.dispose();
 		})
 
-		new Control.Scale({position: 'bottomright'}).addTo(m);
+		// Add PDF button.
+		const ss = new SimpleMapScreenshoter({
+			hideElementsWithSelectors: [
+				'.leaflet-top.leaflet-left',
+				'.leaflet-top.leaflet-right'
+			],
+			preventDownload: true
+		}).addTo(this._map);
+		this._map.on('simpleMapScreenshoter.click', function() {
+			// TODO:  prompt for title and filename, and perhaps block out any interaction w/ a Waiting dialog.
```

Comment:
@sfarber all you!

Reply:
The default behavior is that the screenshot button hides all Leaflet controls.  I think you could create a Waiting... modal for the same effect.